src/main/asciidoc/http.adoc

```diff
@@ -1803,6 +1803,29 @@ use the full URL specified in the request URI:
 {@link examples.HTTPExamples#example60}
 ----
 
+=== Using HA PROXY protocol
+
+https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[HA PROXY protocol] provides a convenient way to safely transport connection
+information such as a client's address across multiple layers of NAT or TCP
+proxies.
+
+HA PROXY protocol can be enabled by setting the option {@link io.vertx.core.http.HttpServerOptions#useProxyProtocol(boolean)}
+and adding the following dependency in your classpath:
```

Comment:
_to_ your classpath ?

Reply:
=== Native Linux Transport

You need to add the following dependency in your classpath:

Reply:
it seems that using "in" is correct